dotnet/src/Agents/Abstractions/Definition/ModelDefinition.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Agents;
+
+/// <summary>
+/// Defines the model to be used by an agent.
+/// </summary>
+[Experimental("SKEXP0110")]
+public sealed class ModelDefinition
+{
+    /// <summary>
+    /// The default API type.
+    /// </summary>
+    private const string DefaultApi = "chat";
```

Comment:
Ok, looking at the `DefaultApi` it's more clear what does `Api` refers to in this context.

A model can be used for than one task right? If this is too disruptive we can keep the one task approach.

Suggest using another name, like, `IList<string> SupportedTasks`

Task seems to be the name for what we are looking here instead of `API`.

AI Foundry:
![image](https://github.com/user-attachments/assets/4a285808-0d6f-48be-b91f-bbe93f52c8e7)

Hugging Face:
![image](https://github.com/user-attachments/assets/ecab2d99-6423-460b-8cef-7551316fddca)


```suggestion
    private const string DefaultSupportedTask = "chat";
```

Reply:
The behaviour is defined by the Foundry Declarative Language single agent specification and we need to ensure our implementation is compatible with the specification.